HWS 42-44/Project10/Project10/ViewController.swift

```diff
@@ -51,7 +66,41 @@ class ViewController: UICollectionViewController, UIImagePickerControllerDelegat
         present(ac,animated: true)
     }
     
-    @objc func addNewPerson() {
+    func removePerson(_ personToRemove: Person) {
+        let ac = UIAlertController(title: "Remove person", message: "Are you sure?", preferredStyle: .alert)
+        let submit = UIAlertAction(title: "Yes", style: .destructive) { [weak self, weak personToRemove] _ in
+            self?.people.removeAll(where: { person in person == personToRemove } )
+            self?.collectionView.reloadData()
+        }
+        ac.addAction(submit)
+        ac.addAction(UIAlertAction(title: "No", style: .cancel))
+        present(ac,animated: true)
+    }
+    
+    @objc func addWithCamera() {
+        let picker = UIImagePickerController()
+        picker.allowsEditing = true
+        picker.delegate = self
+        print(UIImagePickerController.isSourceTypeAvailable(.camera))
+        if(!UIImagePickerController.isSourceTypeAvailable(.camera)){
```

Comment:
W Swift nie dodaje się nawiasów do najbardziej zewnętrznego warunku (btw, brakuje spacji :D) 
Jeśli byłoby kilka warunków, to te warunki rozdzielamy nawiasami np. 
`if (a || b) && (c || d) { }`